config/base/params.env

```diff
@@ -1,4 +1,2 @@
-trustyaiServiceImageName=quay.io/trustyai/trustyai-service
-trustyaiServiceImageTag=latest
-trustyaiOperatorImageName=quay.io/trustyai/trustyai-service-operator
-trustyaiOperatorImageTag=latest
\ No newline at end of file
+trustyaiServiceImage=quay.io/trustyai/trustyai-service:latest
+trustyaiOperatorImage=quay.io/trustyai/trustyai-service-operator:latest
```

Comment:
This might break downstream builds, as far as I remember the replace is done automatically so make sure this change is compatible